extensions/eclipse-che-theia-plugin-ext/src/resource/product.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
why do we need it there if it's embedded json the CheProductServiceImpl ?

Reply:
I also have thought about that. Will remove it.

Reply:
Removed